doc/source/programs/gdal_vector_geom_op_explode_collections.rst

```diff
@@ -0,0 +1,80 @@
+.. _gdal_vector_geom_op_explode_collections_subsubcommand:
+
+================================================================================
+"gdal vector geom-op explode-collections" sub-sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Explode geometries of type collection of a vector dataset.
+
+.. Index:: gdal vector geom-op explode-collections
+
+Synopsis
+--------
+
+.. program-output:: gdal vector geom-op explode-collections --help-doc
+
+Description
+-----------
+
+:program:`gdal vector geom-op explode-collections` produces one feature for
+each geometry in any kind of geometry collection.
+
+For example if a feature contains a geometry ``MULTIPOINT(1 2,3 4)``, two
+corresponding target features will be generated: one with a geometry
+``POINT(1 2)`` and another one with geometry ``POINT(3 4)``
```

Comment:
Clarify if collections are recursively exploded?

Reply:
no, they are not. clarified in doc